### PR TITLE
feat: item family - item variants

### PR DESCRIPTION
**Pull request by pawelKapl**
Adding a way to display item variants - a family - a wrapper that keeps relation between items of one kind.

### CODE REVIEW

File: database/postgresql/src/main/resources/application.properties [hunk 1]
```diff
@@ -3,3 +3,4 @@ quarkus.datasource.username=${QUARKUS_DATASOURCE_USERNAME}
 quarkus.datasource.password=${QUARKUS_DATASOURCE_PASSWORD}
 quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/main_postgres_db
 quarkus.hibernate-orm.physical-naming-strategy=org.hibernate.boot.model.naming.CamelCaseToUnderscoresNamingStrategy
+quarkus.hibernate-orm.log.sql=true
```
Comment (Augustyn) [inline database/postgresql/src/main/resources/application.properties:6]: Let's add this to -dev or -local, or other profiles. The 'default'/production should not log SQLs.

File: quarkus/src/main/java/com/example/javaopencommerce/catalog/ItemController.java [hunk 2]
```diff
@@ -3,28 +3,39 @@
 import com.example.javaopencommerce.PageDto;
 import com.example.javaopencommerce.catalog.dtos.FullItemDto;
 import com.example.javaopencommerce.catalog.dtos.ItemDto;
+import com.example.javaopencommerce.catalog.dtos.ItemFamilyMemberDto;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
```
Comment (Augustyn) [inline quarkus/src/main/java/com/example/javaopencommerce/catalog/ItemController.java:9]: The second one is not required, as it's covered by first import. 
I'm not for wildcard exports... but let's do either: wildcard it or improt explicit

**Review by Augustyn — APPROVED** [review]

File: quarkus/src/main/java/com/example/javaopencommerce/catalog/ItemController.java (hunk 2 above)
Comment (pawelKapl) [reply quarkus/src/main/java/com/example/javaopencommerce/catalog/ItemController.java:9]: Wont agree here, both are needed, wildcard for one package does not mean that it covers all the sub-packages.

File: database/postgresql/src/main/resources/application.properties (hunk 1 above)
Comment (pawelKapl) [reply database/postgresql/src/main/resources/application.properties]: changed!

**Pull request merged**